docs/user-guide.rst

```diff
@@ -29,6 +29,16 @@ that supports OpenID Connect (OIDC).
 Obtaining credentials
 ---------------------
 
+Important: If you accept a credential configuration (credential JSON/File/Stream)
```

Comment:
Maybe this should be "Important Security Notice" or something as a heading / bolded / in red / ?, up to you

Reply:
marked as warning